CRM/Civixero/Invoice.php

```diff
@@ -288,6 +289,11 @@ protected function mapToAccounts($invoiceData, $accountsID) {
       'LineAmountTypes' => $line_amount_types,
       'LineItems' => ['LineItem' => $lineItems],
     ];
+    if (!empty($xeroInvoiceUUID)) {
+      if (!empty($xeroContactUUID)) {
+        $new_invoice['InvoiceID'] = $xeroContactUUID;
+      }
+    }
```

Comment:
This is the only "change". The variable was being passed in but never used, now it will be if available (same as what is done on contact).